mods/common/chrome/lobby-options.yaml

```diff
@@ -48,6 +48,11 @@ Background@LOBBY_OPTIONS_BIN:
 					Width: 150
 					Height: 20
 					Text: Debug Menu
+				Checkbox@BUILDRADIUS_CHECKBOX:
```

Comment:
Tiberian Sun doesn't include the base radius, so can you please duplicate the `lobby-options.yaml` into `mods/ts/chrome` and adjust its `mod.yaml` to use it, the same way d2k does?